### PR TITLE
Remove the old example and port shello to piet-wgsl.

### PR DESCRIPTION
**Pull request by jneem**
The old `gpu` example is failing because it wants `piet-gpu-hal` which no longer exists.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -8,6 +8,7 @@ repository = "https://github.com/linebender/druid"
 readme = "README.md"
 categories = ["os::macos-apis", "os::windows-apis", "gui"]
 edition = "2018"
+resolver = "2"
```
Comment (raphlinus) [inline Cargo.toml:11]: We might consider updating to 2021, but that might be out of scope for this PR.

**Review by raphlinus — APPROVED** [review]
Thanks, glad to see this! It should resolve CI issues and also make for a more relevant experience for people coming to the crate.

File: Cargo.toml (hunk 1 above)
Comment (derekdreery) [reply Cargo.toml:11]: When playing around I updated to 2021 and it "just worked".

**Comment by jneem** [comment]
CI didn't complain about the edition bump, so I guess it's probably ok...

**Pull request merged**